src/Symfony/Component/Runtime/autoload.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Component\Runtime\SymfonyRuntime;
+
+if (true === require_once dirname(__DIR__, 2).'/autoload.php') {
+    return;
+}
+
+if (!isset($_SERVER['APP_RUNTIME'])) {
+    $runtime = SymfonyRuntime::boot($_SERVER['APP_RUNTIME_OPTIONS'] ?? []);
+} elseif (is_string($runtime = $_SERVER['APP_RUNTIME'])) {
```

Comment:
maybe we can improve check with `is_a`, asserting it's subclass of `RuntimeInterface` here ?

Reply:
I'd prefer not, it's the hot path.

Reply:
But in this case we really do not need interface. Duck typing does all the job, since you are not requiring it anywhere. any class having proper `resolve` and `start` (or whatether their names are now) will suit